BridgeAppSDKSample/StudyOverviewViewController.swift

```diff
@@ -31,21 +31,16 @@
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 //
 
-import ResearchKit
+import UIKit
+import BridgeAppSDK
 
-public protocol SBAConsentReviewOptions: class {
-    var requiresSignature: Bool { get }
-    var requiresName: Bool { get }
-}
+class StudyOverviewViewController: UIViewController {
 
-extension NSDictionary: SBAConsentReviewOptions {
-    
-    public var requiresSignature: Bool {
-        return self["requiresSignature"] as? Bool ?? true
+    @IBAction func signUpTapped(sender: AnyObject) {
+        // TODO: implement
     }
     
-    public var requiresName: Bool {
-        return self["requiresName"] as? Bool ?? true
+    @IBAction func loginTapped(sender: AnyObject) {
+        // TODO: implement
```

Comment:
is this (and the above TODO) meant as a comment to developers starting from the sample app? otherwise name/date